### DISCUSSION TITLE
Ss-iptv

### DISCUSSION THREAD

**Discussion by mg1067** [iptv-org/community#215] [Q&A] [answered]
Hi
I used the Playlist country in kodi and this runs perfekt.
Now i Like to use ss-iptv on my smart-tv.
I can not activate this list . Is there a Other way ?
Do i Need Convert the list to use?

Thanks ...
micky1067

**Comment by thedarkdragon11** [comment, marked as answer]
I believe it's possible but SS-IPTV doesn't support non-HLS links and custom user agents.

**Reply by mg1067** [reply]
Thansk for your reply. So I must look what I must done. 
Maybe someone have a idea.
